core/src/main/scala/kafka/admin/ConfigCommand.scala

```diff
@@ -246,6 +247,18 @@ object ConfigCommand extends Config {
 
   private[admin] def parseConfigsToBeAdded(opts: ConfigCommandOptions): Properties = {
     val props = new Properties
+    if (opts.options.has(opts.addConfigFile)) {
+      val file = opts.options.valueOf(opts.addConfigFile)
+      val inputStream = try new FileInputStream(file) catch {
+        case _: FileNotFoundException =>
+          throw new IllegalArgumentException(s"No such file or directory: $file")
```

Comment:
nit: I would remove the `or directory` here as we don't support directories at all.